src/styles/product-card.scss

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`